src/VolleyManagement.WebClient/spa/app/app.component.html

```diff
@@ -1,3 +1,3 @@
 Volley-Management App
 
-<schedule [scheduleId]="3"></schedule>
\ No newline at end of file
+<schedule [scheduleId]="13"></schedule>
```

Comment:
If it is hardcoded, isn't it better to move it to app constants or environment configs?

Reply:
It will be replaced later with router